geoportal/c2cgeoportal_geoportal/scaffolds/create/bin/eval-templates

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+find / -name *.tmpl -print | while read file
+do
+    envsubst < ${file} > ${file%.tmpl}
+done
```

Comment:
Arn't you missing the `exec "$@"`?

Reply:
To do what? I don't have anything to exec ...

Reply:
It's not used as an entrypoint?

Reply:
Yes, but in the config container, than he hasn't any other "entry point"...

Reply:
By not putting an exec in there, you make the life of the dev wanting to debug this image harder... anyway, not a big deal...